### PR TITLE
Add compilemessages to Ansible

### LINKED ISSUE #367: Ansible playbook does not restart the gunicorn workers at the end
Manual fix to apply: `supervisorctl` then `restart <the mangaki app>` (which is `mangaki_staging` on beta) then Ctrl + D.

### LINKED ISSUE #373: Ansible playbook does not compilemessages
(for i18n)

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Partial fix, I'm pretty sure that we need to reconsider how we should `compilemessages` knowing that we support multi type of installation (so that `mangaki` is not always importable and so on).

Easy way: we should just compile `.mo` locally.

Also, it definitely fixes #367, fixes #373 !

**EDIT: Depends on #474.**

### CODE REVIEW

File: mangaki/mangaki/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1,114 +1,105 @@
-# Mangaki's translation in English.
-# Copyright (C) 2017 Mangaki
-# This file is distributed under the same license as the Mangaki package.
-# Jill-Jênn Vie <vie@jill-jenn.net>, 2017.
+# SOME DESCRIPTIVE TITLE.
```
Comment (Elarnon) [inline mangaki/mangaki/locale/en/LC_MESSAGES/django.po]: Why those changes?

**Comment by Elarnon** [comment]
The ansible aspect looks OK but it also looks like this is *removing* a bunch of translations, which is probably not what we want.

**Comment by jilljenn** [comment]
- Il faut que l'anglais soit la langue par défaut.
- Ryan doit remettre les traductions.

File: provisioning/roles/mangaki_source/tasks/main.yml [hunk 2]
```diff
@@ -71,8 +77,10 @@
   register: project_install
 
 - name: Request service reloading.
-  set_fact:
-    mangaki_needs_restart: true
+  debug:
+    msg: Project contents has changed, notifying a restart.
```
Comment (jilljenn) [inline provisioning/roles/mangaki_source/tasks/main.yml]: have*

**Pull request merged**